<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="桂宝元" name="mytest" tests="3" failures="1" timestamp="13 Aug 2018 08:41:29 GMT" time="0.053" errors="0">
  <testcase name="a" time="0.021" classname="test.testng.parralel.MethodP">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
	at test.testng.parralel.MethodP.a(MethodP.java:33)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase name="f" time="0.003" classname="test.testng.parralel.NewTest"/>
  <testcase name="f" time="0.001" classname="test.testng.parralel.MethodP">
    <skipped/>
  </testcase> <!-- f -->
</testsuite> <!-- mytest -->
